datahub-web-react/src/app/entity/glossaryTerm/preview/__tests__/Preview.test.tsx

```diff
@@ -0,0 +1,15 @@
+import { render } from '@testing-library/react';
+import React from 'react';
+import TestPageContainer from '../../../../../utils/test-utils/TestPageContainer';
+import { Preview } from '../Preview';
+
+describe('Preview', () => {
+    it('renders', () => {
+        const { getByText } = render(
+            <TestPageContainer>
+                <Preview urn="urn:li:tag:abc-sample-tag" name="name" definition="definition" owners={null} />
```

Comment:
why is the glossaryterm test accepting a tag urn?

Reply:
Fixed as part of https://github.com/linkedin/datahub/pull/2538